tox.ini

```diff
@@ -14,7 +14,8 @@ setenv =
     COLUMNS = 180
     PIP_INDEX_URL = {env:PIP_INDEX_URL:https://pypi.anaconda.org/scipy-wheels-nightly/simple}
     PIP_EXTRA_INDEX_URL = {env:PIP_EXTRA_INDEX_URL:https://pypi.org/simple}
-    PYTEST_COMMAND = pytest --pyargs plasmapy --durations=5 -n=auto --dist=loadfile
+    PYTEST_COMMAND = pytest --pyargs plasmapy --durations=5 -n=auto --dist=loadfile --tb=short
```

Comment:
The `--tb=short` makes it so that the traceback reports are shorter.  Without it, the default is to print the entire function for the first and last error, which can make it harder to navigate pytest output.